sklearn/model_selection/tests/test_validation.py

```diff
@@ -1666,16 +1666,6 @@ def test_fit_and_score_failing():
     assert_warns_message(FitFailedWarning, warning_message, _fit_and_score,
                          *fit_and_score_args, **fit_and_score_kwargs)
 
-    # check if warning was raised, with default error_score argument
-    warning_message = ("From version 0.22, errors during fit will result "
-                       "in a cross validation score of NaN by default. Use "
-                       "error_score='raise' if you want an exception "
-                       "raised or error_score=np.nan to adopt the "
-                       "behavior from version 0.22.")
-    with pytest.raises(ValueError):
```

Comment:
I think we still want the `with pytest.raises(ValueError):` here.

The `assert_warns_message` is the only part that should be removed:

```py
with pytest.raises(ValueError):
    _fit_and_score(*fit_and_score_args)
```

Reply:
Thanks for the feedback ! Somehow it seems that I still get a test failure from `test_fit_and_score_failing()`, could it be related due to this structure in `_validation.py` ?

```python
except Exception as e:
    # Note fit time as time until error
    fit_time = time.time() - start_time
    score_time = 0.0
    if error_score == 'raise':
        raise
    elif isinstance(error_score, numbers.Number):
        if is_multimetric:
            test_scores = dict(zip(scorer.keys(),
                               [error_score, ] * n_scorers))
            if return_train_score:
                train_scores = dict(zip(scorer.keys(),
                                    [error_score, ] * n_scorers))
        else:
            test_scores = error_score
            if return_train_score:
                train_scores = error_score
        warnings.warn("Estimator fit failed. The score on this train-test"
                      " partition for these parameters will be set to %f. "
                      "Details: \n%s" %
                      (error_score, format_exception_only(type(e), e)[0]),
                      FitFailedWarning)
    else:
        raise ValueError("error_score must be the string 'raise' or a"
                         " numeric value. (Hint: if using 'raise', please"
                         " make sure that it has been spelled correctly.)")
```

Reply:
My bad, I did not realize that the previous default would also raise ValueError.

You can remove it as you previously did.

(in the future please avoid mentioning with @username in the git commit messages, because it will send notifications everytime you commit which is not necessarily wanted ;) )

Reply:
Got it - apologies for that ! Hopefully this will be the last commit in this PR and then the notifications will stop, noted for future reference.